src/Themes/NexusMods.Themes.NexusFluentDark/NexusFluentDarkTheme.axaml

```diff
@@ -5,11 +5,11 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceInclude Source="/Resources/ResourceIndex.axaml"/>
+                <ResourceInclude Source="avares://NexusMods.App.UI/Pages/LibraryPage/TreeDataGridLibraryResources.axaml"/>
```

Comment:
I don't think this should be here:
- It is referencing a resource that isn't in the theme project
- It logically does not belong in the Theme project
- The Library Resources are available everywhere implicitly, there is no line connecting the `TreeDataGrid` using the resources back to the resource file, which makes it hard to find it.

We can instead name the LibraryResources using a Key, and then import them directly in the `TreeDataGrid`s that actually use them, so that there is a clear and explicit path from the usage point to the templates, and this reference isn't here in the Themes project.

Reply:
Where should the resource include statement go? It has to be somewhere.

Reply:
It can't be imported directly in the View file using it then?

Reply:
No, we have two views that need it.

Reply:
It can't be imported in two files?

Reply:
It can, but I think it's a bad practice. We've had similar issues when we first created the theme project as well.

Reply:
Ok, I think we should put it in `App.axaml` under `<Application.Resources>` then.
It's where Avalonia docs says to put it.